views/index.hbs

```diff
@@ -1,11 +1,17 @@
-<h1>This is the homepage</h1>
+<div id="container">
+
+    <img id="background-img" src="/images/spotify-background.jpeg">
```

Comment:
At the moment, when I view your app in the browser, the container element is not full screen.

Here I would suggest rather than creating an image tag, just add a background image to your div with id 'container'. Also set that container element to the full width and height of the page, so for example 
```


#container {
  background-image: url("/images/spotify-background.jpeg");
  background-position: center;
  background-size: cover;
   /* position: relative; */
    text-align: center;
    height: 100vh;
    width: 100vw;
}
```